PurrSoft/PurrSoft.Application/Models/VolunteerDto.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace AlbumStore.Application.Models;
+
+public class VolunteerDto
+{
+    public string UserId { get; set; }
+    public string StartDate { get; set; }
+    public string? EndDate { get; set; }
+    public string Status { get; set; }
+    public string Tier { get; set; }
+    public string AssignedArea { get; set; }
+    // public virtual ICollection<Shift> Shifts { get; set; }
```

Comment:
Here they won't be virtual. Virtual are in the classes that are handled by entity framework